Expensify/Bedrock-PHP
# [hold]Clear stuck jobs on BWM restart

We're continuously getting jobs that get stuck running and never update. The best way to deal with that from the standpoint of Bedrock-PHP is to clear out stuck jobs whenever it is restarted, since this happens frequently enough that we shouldn't ever gather too many stuck jobs.

Hold for packaging.

# Issue
https://github.com/Expensify/Expensify/issues/61961

# Tests
1. Insert jobs into DB with `ended=null`
2. Start bwm with `--enableLoadHandler`
3. Check that the initial jobs were deleted.
